cves/kernel/CVE-2017-16913.yml

```diff
@@ -452,7 +452,7 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: The main mistake made for this vulnerability was a coding mistake. Fundamentally, the design of USB/IP is robust. However, certain necessary checks were not made resulting in vulnerable code. 
```

Comment:
Reiterate the check that was not made, in this case the size of the size fields, just to show the simplicity of the fix.